lix/packages/sdk/src/file-handlers.ts

```diff
@@ -141,30 +160,48 @@ export async function handleFileChange(args: {
 				// working change exists but is identical to previously committed change
 				if (previousChange) {
 					previousCommittedDiff = await pluginDiffFunction?.[diff.type]?.({
-						old: previousChange?.value,
+						old: previousChange.value,
 						neu: diff.neu,
 					});
 
-					if (previousCommittedDiff?.length === 0) {
-						// drop the change because it's identical
+					if (previousCommittedDiff.length === 0) {
```

Comment:
Intersting that tsc doesn't complain about this...

we asign it to
`await pluginDiffFunction?.[diff.type]?.({
						old: previousChange.value,
						neu: diff.neu,
					});`
which can be undefined if pluginDiffFunction is undefined or it ha no proeperty named like the changes type. This code has changed for that reason in lix-integration